src/include/daos_srv/vos.h

```diff
@@ -425,6 +425,35 @@ vos_obj_fetch(daos_handle_t coh, daos_unit_oid_t oid, daos_epoch_t epoch,
 	      uint64_t flags, daos_key_t *dkey, unsigned int iod_nr,
 	      daos_iod_t *iods, d_sg_list_t *sgls);
 
+/**
+ * Fetch values for the given keys and their indices.
+ * If output buffer is not provided in \a sgl, then this function returns
+ * the directly accessible addresses of record data, upper layer can directly
+ * read from these addresses (rdma mode).
+ *
+ * TODO: add more detail descriptions for punched or missing records.
+ *
+ * \param coh	[IN]	Container open handle
```

Comment:
Effort was made to update vos_ts.h to move [IN] to before param name. Is there a reason to have them before vs after? Also uper vs lowercase? No big deal, just wondering about consistency.

Reply:
\param[in] is correct for Doxygen.   So I'm opportunistically updating files.